src/browser/components/icons/Icons.tsx

```diff
@@ -22,35 +22,35 @@ import React from 'react'
 import { Icon } from 'semantic-ui-react'
 import { keyframes, css } from 'styled-components'
 import { IconContainer } from './IconContainer'
-import ratingStar from 'icons/rating-star.svg'
-import databaseCheck from 'icons/database-check.svg'
-import cog from 'icons/cog.svg'
+import addCircle from 'icons/add-circle.svg'
+import appWindowCode from 'icons/app-window-code.svg'
+import arrowLeft from 'icons/arrow-left.svg'
+import arrowRight from 'icons/arrow-right.svg'
+import buttonRefreshArrow from 'icons/button-refresh-arrow.svg'
+import cannyFeedback from 'icons/canny-feedback.svg'
+import cannyNotifications from 'icons/canny-notifications.svg'
+import close from 'icons/close.svg'
 import cloudCheck from 'icons/cloud-check.svg'
 import cloudRemove from 'icons/cloud-remove.svg'
-import neo4j from 'icons/neo-world.svg'
-import pin from 'icons/pin.svg'
-import close from 'icons/close.svg'
-import expand01 from 'icons/expand-01.svg'
-import shrink from 'icons/shrink.svg'
-import buttonRefreshArrow from 'icons/button-refresh-arrow.svg'
+import cog from 'icons/cog.svg'
+import databaseCheck from 'icons/database-check.svg'
 import downloadBottom from 'icons/download-bottom.svg'
-import table01 from 'icons/table-01.svg'
-import Text201 from 'icons/Text2-01.svg'
-import appWindowCode from 'icons/app-window-code.svg'
-import arrowLeft1 from 'icons/arrow-left-1.svg'
-import arrowRight1 from 'icons/arrow-right-1.svg'
-import skipPrev from 'icons/skip-prev.svg'
+import expand from 'icons/expand.svg'
 import file from 'icons/file.svg'
-import save_favorite from 'icons/save_favorite.svg'
-import run_icon from 'icons/run_icon.svg'
-import stop_icon from 'icons/stop_icon.svg'
+import folderEmpty from 'icons/folder-empty.svg'
 import help from 'icons/help.svg'
+import navigationMenuVertical from 'icons/navigation-menu-vertical.svg'
+import neo4j from 'icons/neo-world.svg'
 import newFolder from 'icons/folder-add.svg'
-import folder from 'icons/folder-empty.svg'
-import addCircle from 'icons/add-circle.svg'
-import navIcon from 'icons/navigation-menu-vertical.svg'
-import cannyFeedback from 'icons/canny-feedback.svg'
-import cannyNotifications from 'icons/canny-changelog.svg'
+import pin from 'icons/pin.svg'
+import ratingStar from 'icons/rating-star.svg'
+import runIcon from 'icons/run-icon.svg'
+import saveFavorite from 'icons/save-favorite.svg'
+import shrink from 'icons/shrink.svg'
+import skipPrev from 'icons/skip-prev.svg'
+import stopIcon from 'icons/stop-icon.svg'
+import table from 'icons/table.svg'
+import text from 'icons/text.svg'
```

Comment:
This new sort order makes my inner OCD very happy, thanks for doing this! :-)

Reply:
Really easy to do in vim as well, with it the lines selected all I had to to was type :sort 💯 kinda want to start doing it every now and then on messy imports